tests/tests/swfs/avm2/click_invisible/Test.as

```diff
@@ -0,0 +1,46 @@
+﻿package {
+	import flash.display.MovieClip;
+	import flash.display.Sprite;
+	import flash.display.Shape;
+	import flash.geom.Rectangle;
+	import flash.events.MouseEvent;
+	import flash.display.BitmapData;
+	import flash.display.Bitmap;
+	import flash.text.TextField;
+
+	public class Test extends MovieClip {
+		public function Test() {
+		    var shapeContainer = new Sprite();
+			shapeContainer.name = "MyShapeContainer";
+			var shape = new Shape();
+			shape.graphics.beginFill(0xFF0000);
+			shape.graphics.drawRect(0, 0, 100, 100);
+			shape.graphics.endFill();
+			shapeContainer.addChild(shape);
+			this.addChild(shapeContainer);
+			
+			var bitmapContainer = new Sprite();
+			bitmapContainer.name = "MyBitmapContainer";
+			var bitmapData = new BitmapData(120, 120, false, 0);
```

Comment:
(Very minor nit, consider ignoring: Is `0` transparent black or opaque black? Maybe write in hex-code?).